Gemfile.lock

```diff
@@ -17,6 +17,7 @@ GEM
     ethon (0.9.1)
       ffi (>= 1.3.0)
     ffi (1.9.14)
+    ffi (1.9.14-x64-mingw32)
```

Comment:
I'll fix.  Is there a way to avoid this in the future by chance?  Seems like the gemfile gets modified as a result of running doc generation - should it just be in the .gitignore ?